windows/win_user.ps1

```diff
@@ -101,7 +101,8 @@ If ($account_locked -ne $null) {
 
 $groups = Get-Attr $params "groups" $null
 If ($groups -ne $null) {
-    If ($groups.GetType().Name -eq "String") {
+    $groups_type = $groups.GetType().Name
+    If ($groups_type -eq "String[]" -or $groups_type -eq "String") {
         [string[]]$groups = $groups.Split(",")
```

Comment:
I don't have my env to test now, but I have seen that sometimes, the GetType().name is String and sometimes it's String[].

I have to re-do it, but basically there are four cases :

- name: Set Windows users
  win_user: name=mylogin password=mypassword groups_action=replace groups=["Users", "Administrators"] state=present

- name: Set Windows users
  win_user: name=mylogin password=mypassword groups_action=replace groups=["Users"] state=present
  
- name: Set Windows users
  win_user: name=mylogin password=mypassword groups_action=replace groups=Users,Administrators state=present

- name: Set Windows users
  win_user: name=mylogin password=mypassword groups_action=replace groups=Users state=present